src/app/containers/Metadata/index.jsx

```diff
@@ -32,6 +32,10 @@ const pageTypeMetadata = {
     schemaOrg: 'RadioChannel',
     openGraph: 'website',
   },
+  FIX: {
+    schemaOrg: 'WebPage',
+    openGraph: 'website',
+  },
```

Comment:
Do we also need to declare `MAP` here?

Reply:
We don't at the moment.

Reply:
The media pod will be doing that as part of https://github.com/bbc/simorgh/issues/3940